config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -4476,6 +4472,10 @@ spec:
                     type: string
                   maxItems: 100
                   type: array
+                advancedConfig:
+                  description: Specify any rabbitmq advanced.config configurations
+                  maxLength: 2000
```

Comment:
Is this limit required? 2000 characters can be exceeded in environments with extensive LDAP queries and OAuth 2 settings, for example.

Reply:
It's not required. There is a hard limit in kubernetes that all objects are stored in etcd, and etcd has a limit of 1MB per object which translate to `1048576` characters. I will update the limit to `100000`. Thanks for pointing it out Michael!